Docker/ParallelZone.Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Copyright 2023 NWChemEx-Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:20.04
+LABEL maintainer="Yu Zhang <yzhang23@ameslab.gov>" \
```

Comment:
Do you need the email? If not, I'd just change this to `"NWChemEx-Project"`

Reply:
Sorry, forget to remove my info in the docker file. Will update this.